modules/nw-cudn-localnet.adoc

```diff
@@ -0,0 +1,79 @@
+//module included in the following assembly:
+//
+// * networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-cudn-localnet_{context}"]
+= Creating a ClusterUserDefinedNetwork CR for Localnet
+
+A `Localnet` topology connects the secondary network to the physical underlay. This enables both east-west cluster traffic and access to services running outside the cluster, but it requires additional configuration of the underlying Open vSwitch (OVS) system on cluster nodes.
+
+.Prerequisites
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+* You have created and configured the Open vSwitch (OVS) bridge mapping to associate the physical network with the interface label created by OVS. For more information, see "Configuration for a localnet switched topology".
+
+.Procedure
+
+. Create a request for `Localnet` topology type cluster-wide user-defined network:
+
+.. Create a YAML file, such as `cluster-udn-localnet.yaml`, to define your request for a `Layer2` topology as in the following example:
++
+[source, yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: ClusterUserDefinedNetwork
+metadata:
+  name: <cudn_name> # <1>
+spec:
+  namespaceSelector: # <2>
+    matchLabels: # <3>
+    - "<example_namespace_one>":"" # <4>
+    - "<example_namespace_two>":"" # <4>
+  network: # <5>
+    topology: Localnet # <6>
+    localnet: # <7>
+      role: Secondary # <8>
+      physicalNetworkName: test
+      ipam: {lifecycle: Persistent}
+      subnets: ["192.168.0.0/16", "2001:dbb::/64"] # <9>
+----
+<1> Name of your `ClusterUserDefinedNetwork` CR.
+<2> A label query over the set of namespaces that the cluster UDN CR applies to. Uses the standard Kubernetes `MatchLabel` selector. Must not point to `default` or `openshift-*` namespaces.
+<3> Uses the `matchLabels` selector type, where terms are evaluated with an `AND` relationship.
+<4> Because the `matchLabels` selector type is used, provisions namespaces matching both `<example_namespace_one>` _and_ `<example_namespace_two>`.
+<5> Describes the network configuration.
+<6> The `topology` field describes the network configuration; accepted values are `Layer2`, `Layer3`, and `Localnet`. Specifying a `Localnet` topology type creates one logical switch that is shared by all nodes.
```

Comment:
aren't we providing documentation for the localnet related use-cases ? Shouldn't we say we need to use `Localnet` here ?

Reply:
Agree